### DISCUSSION TITLE
Proper way to take down a misskey from fediverse

### DISCUSSION THREAD

**Discussion by spirillen** [misskey-dev/misskey#14973] [Q&A]
I have been looking around the net to see if I could find any documentation, on how to discontinue a misskey instance from the fediverse. however, it seems like none official documentation are public available

![Image](https://github.com/user-attachments/assets/33e122d3-a465-45bc-bc03-a6a60a677ca5)

![Image](https://github.com/user-attachments/assets/9e2e8eda-d43c-413e-aaaa-9a5ba6681c7f)

This makes it impossible for anyone to read anything, so please explain how the procedures are to disconnect and set a instance offline from the fediverse. The server was installed via the bash scipt

**Comment by kakkokari-gtyih** [comment]
Although it's not required, it is recomended to return 410 (Gone) to tell other servers that your server is no longer in service.

1. Stop the misskey server
2. Start the web server (e.g. nginx, apache) instead, and return 410 for every route

**Reply by spirillen** [reply]
So unlike mastodon and paloma, you do not have a build in feature to tell other fediverse servers that this instance are leaving?

**Reply by anatawa12** [reply]
No, We don't have feature to tell that the instance are leaving in misskey. You have to use some other software to send 410 Gone.
